--- /tmp/pacman.conf.orig	2017-10-16 16:50:33.539971666 +0900
+++ /etc/pacman.conf	2017-10-16 16:51:08.534468909 +0900
@@ -35,6 +35,7 @@
 #TotalDownload
 CheckSpace
 #VerbosePkgLists
+ILoveCandy
 
 # By default, pacman accepts packages signed by keys that its local keyring
 # trusts (see pacman-key and its man page), as well as unsigned packages.
@@ -90,8 +91,8 @@
 #[multilib-testing]
 #Include = /etc/pacman.d/mirrorlist
 
-#[multilib]
-#Include = /etc/pacman.d/mirrorlist
+[multilib]
+Include = /etc/pacman.d/mirrorlist
 
 # An example of a custom package repository.  See the pacman manpage for
 # tips on creating your own repositories.
