--- system-login.orig	2017-12-24 06:39:37.514170539 +0900
+++ system-login	2017-12-24 11:38:40.496676054 +0900
@@ -1,6 +1,7 @@
 #%PAM-1.0
 
-auth       required   pam_tally.so         onerr=succeed file=/var/log/faillog
+auth       required   pam_tally2.so deny=3 unlock_time=600 onerr=succeed
+#auth       required   pam_tally.so         onerr=succeed file=/var/log/faillog
 auth       required   pam_shells.so
 auth       requisite  pam_nologin.so
 auth       include    system-auth
